client/scripts/views/components/chain_icon.tsx

```diff
@@ -68,7 +71,7 @@ export class TokenIcon implements m.ClassComponent<TokenIconAttrs> {
     const { onclick, size = 32, token } = vnode.attrs;
 
     return (
-      <div class={`TokenIcon ${onclick ? 'onclick' : ''}`}>
+      <div class={`TokenIcon${onclick ? ' onclick' : ''}`}>
```

Comment:
if we're gonna be particular, i prefer the space in the literal and not the conditional: 
`TokenIcon ${onclick ? 'onclick' : ''}`